src/test/java/AndreeaG/tests/tema6/DemoQaWindow.java

```diff
@@ -0,0 +1,114 @@
+package AndreeaG.tests.tema6;
+
+import AndreeaG.driver.BrowserManager;
+import AndreeaG.utils.FileUtils;
+import org.openqa.selenium.By;
+import org.openqa.selenium.NoSuchElementException;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+
+import java.util.Set;
+
+public class DemoQaWindow {
+
+    static ChromeDriver driver = null;
+
+    public static void main(String[] args) {
+        navigateToDemoQAWindowPage();
+        clickOnNewTabButton();
+        clickOnNewWindow();
+        try {
+            clickOnNewWindowMessage();
+        } catch (NoSuchElementException e) {
+            ;
+            FileUtils.takeScreenshot(driver, "no such element");
+        } finally {
+            driver.close();
```

Comment:
Use closeBrowser() method defined at the end of the class instead of driver.close(). Thanks!